core/infra/connection/connection.grpc.pb.h

```diff
@@ -0,0 +1,104 @@
+// Generated by the gRPC protobuf plugin.
+// If you make any local change, they will be lost.
+// source: connection.proto
+#ifndef GRPC_connection_2eproto__INCLUDED
+#define GRPC_connection_2eproto__INCLUDED
+
+#include "connection.pb.h"
+
+#include <grpc++/impl/codegen/async_stream.h>
+#include <grpc++/impl/codegen/async_unary_call.h>
+#include <grpc++/impl/codegen/proto_utils.h>
+#include <grpc++/impl/codegen/rpc_method.h>
+#include <grpc++/impl/codegen/service_type.h>
+#include <grpc++/impl/codegen/status.h>
+#include <grpc++/impl/codegen/stub_options.h>
+#include <grpc++/impl/codegen/sync_stream.h>
+
+namespace grpc {
+class CompletionQueue;
+class Channel;
+class RpcService;
+class ServerCompletionQueue;
+class ServerContext;
+}  // namespace grpc
+
+namespace connection_object {
+
+class IrohaConnection GRPC_FINAL {
+ public:
+  class StubInterface {
+   public:
+    virtual ~StubInterface() {}
+    virtual ::grpc::Status Operation(::grpc::ClientContext* context, const ::connection_object::ConsensusEvent& request, ::connection_object::StatusResponse* response) = 0;
+    std::unique_ptr< ::grpc::ClientAsyncResponseReaderInterface< ::connection_object::StatusResponse>> AsyncOperation(::grpc::ClientContext* context, const ::connection_object::ConsensusEvent& request, ::grpc::CompletionQueue* cq) {
+      return std::unique_ptr< ::grpc::ClientAsyncResponseReaderInterface< ::connection_object::StatusResponse>>(AsyncOperationRaw(context, request, cq));
+    }
+  private:
+    virtual ::grpc::ClientAsyncResponseReaderInterface< ::connection_object::StatusResponse>* AsyncOperationRaw(::grpc::ClientContext* context, const ::connection_object::ConsensusEvent& request, ::grpc::CompletionQueue* cq) = 0;
+  };
+  class Stub GRPC_FINAL : public StubInterface {
+   public:
+    Stub(const std::shared_ptr< ::grpc::ChannelInterface>& channel);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class 'Stub' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4621381380/issues/source?bid=3712501&fileBranchId=3957673#l42)